src/SIL.Machine.AspNetCore/Services/DistributedReaderWriterLockFactory.cs

```diff
@@ -31,7 +31,7 @@ public IDistributedReaderWriterLock Create(string id)
 
     public async Task<bool> DeleteAsync(string id, CancellationToken cancellationToken = default)
     {
-        RWLock? rwLock = await _locks.DeleteAsync(id, cancellationToken);
+        RWLock? rwLock = await _locks.DeleteAsync(e => (string)e.Id == id, cancellationToken);
```

Comment:
Why did you change this to a lambda expression?

Reply:
It didn't work with the id just as a string - it threw an error.  I looked into it and this was the clearest resolution.  It works.